### ISSUE TITLE
Landing Page

### ISSUE THREAD

**Issue by ghost** [theupweb/idea-lab#7]
[Check the Design of the Landing Page ](https://github.com/theupweb/idea-lab)

Comment to this thread so that I can give you the resources (files, icons, SVG, png, logo, etc.)

**Expected Skills**: ReactJS, HTML, CSS and Javacript.

**Comment by ankita413** [comment]
Okay. Sure pls share the resources

**Comment by himanshu007-creator** [comment]
i would like to work on it, please assign to me

**Comment by ghost** [comment]
I would like to work on that

**Comment by shrushti2000** [comment]
I would like to work on this issue.

**Comment by Priyanshu-official** [comment]
@JoshuaPoddoku I also Would like to contribute

**Comment by Amit0412** [comment]
Hello sir. If there is anything related to currently HTML/CSS kindly assign to me. I am learning IS, REACTJS, JQUERY. I will be comfortable in JavaScript  and other stuff in 1-2 weeks. So if any work related to now HTML/CSS I would love to contribute. Thank you

**Comment by devmadhavvv** [comment]
I'm interested

**Comment by nishant2525** [comment]
Ok please share the resources.

**Comment by kanishkaa24** [comment]
I want to work on this project.

**Comment by antaramunshi** [comment]
i would like to work on this

**Comment by Priyanshu-official** [comment]
@JoshuaPoddoku I will start working on this

**Comment by Priyanshu-official** [comment]
@JoshuaPoddoku Tell me We should break The landing page into a component like Nav and home??
or Landing page Issue will have both

**Comment by ghost** [comment]
Yeah, we should break @Priyanshu-official to take the advantage of react 👍

**Comment by ankita413** [comment]
@JoshuaPoddoku so if we are going to break the page should we start building the individual components. like navbar,footer,mid-content page etc. this individual components can be built by different contributors or same contributor

**Comment by Priyanshu-official** [comment]
@JoshuaPoddoku ya great So let's do this. I am fine now 
We have already design I am work on that

**Comment by Priyanshu-official** [comment]
@JoshuaPoddoku Any one Want to Work let me know to this thread...

**Comment by himanshu007-creator** [comment]
> Any one Want to Work let me know to this thread...

Interested!!

**Comment by kanishkaa24** [comment]
> Any one Want to Work let me know to this thread...

I would love to work on this. What are the requirements?

**Comment by Priyanshu-official** [comment]
@kanishkaa24 pls go through the readme file you will get all info

**Comment by kanishkaa24** [comment]
> @kanishkaa24 pls go through the readme file you will get all info

I would love to work on this. Please assign me.

**Comment by Priyanshu-official** [comment]
@kanishkaa24 You can Work on **Navigation** component  according the design Also remember we are using [tailwindcss](https://tailwindcss.com/) 
you have 24hrs for making PR go head.
we are not assign anyone you can make PR it will accepted.

**Comment by Priyanshu-official** [comment]
> > Any one Want to Work let me know to this thread...
> 
> Interested!!

will let you know When new component needed

**Comment by Priyanshu-official** [comment]
@himanshu007-creator  Will let you know check issue section

**Comment by Priyanshu-official** [comment]
@kanishkaa24 pls Let me Know if you are Working on Nav

**Comment by kanishkaa24** [comment]
> @kanishkaa24 pls Let me Know if you are Working on Nav

You mean the navigation bar in the Readme.md, Right?

**Comment by Priyanshu-official** [comment]
@kanishkaa24 yes

**Comment by ghost** [comment]
@all landing page needs to be updated as per UI.

Let me know if you need any help
